### DISCUSSION TITLE
Disabling browser extensions

### DISCUSSION THREAD

**Discussion by ak21edp** [SafeExamBrowser/seb-win-refactoring#817] [Q&A]
A client was testing an SEB config file we built for security vulnerabilities in the middle east and was able to use Google Translate within the SEB application (windows device). What is the easiest way to disable browser extensions assuming that is what is causing this issue? They are taking and English test so while the extension may be helpful for day to day use it has to be blocked for testing. 

I tried to reproduce this issue but was unsuccessful. I also reviewed all documentation I could find.

See images below. This could be an issue with the settings we are deploying but I was not sure what additional steps to take.

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/153239688/304eaede-12fa-4da0-8dff-c8913f4524c8)
![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/153239688/9a357cbc-c991-4e96-b001-b45a088d6e82)

**Comment by dbuechel** [comment]
Terribly sorry, but without clear instructions of a reliable way to reproduce the issue there is nothing we can do here. Browser extensions should not work in SEB, at least is there no intention of us to support them. Could it not even be that Google Translate was somehow embedded in the web page / LMS where the exam was taking place?

**Reply by ak21edp** [reply]
Thank you for getting back to me Damian. After further investigation, it looks like that might be the case. To be frank I am not a technical wiz (yet) but I have attached the response our engineer was able to put together after finally being able to reproduce the issue. I will speak with him today to figure out what settings we may need to change in th configuration to disable; assuming it can be done on our end.

![image](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/153239688/3190e5cf-7d96-453c-b413-863b2b2bf545)

**Reply by dbuechel** [reply]
Yes, that definitely looks like a JavaScript script which embeds Google Translate in your web application.
